t/node/global-rule.t

```diff
@@ -120,7 +120,34 @@ GET /hello
 
 
 
-=== TEST 6: delete global rule
+=== TEST 6: not limited
+--- request
+GET /apisix/nginx_status
+--- error_code: 200
+--- no_error_log
+[error]
+
+
+
+=== TEST 7: not limited
```

Comment:
Why add a same test case?

Reply:
@tokers  because it's testing the `limit-count` plugin here.